x-pack/plugins/lens/public/metric_visualization/visualization.tsx

```diff
@@ -59,6 +60,9 @@ const toExpression = (
         function: 'lens_metric_chart',
         arguments: {
           title: [attributes?.title || ''],
+          size: [state?.size || ''],
+          titlePosition: [state?.titlePosition || 'top'],
```

Comment:
Not opposed to default to "top" for new visualizations, but the old default was "bottom". So if we want to keep the default value here we have to add a migration to add in the `bottom` for the existing ones.